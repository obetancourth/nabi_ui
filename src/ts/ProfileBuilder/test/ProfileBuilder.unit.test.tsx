import * as React from 'react';
import { ProfileBuilder } from '../';
import { shallow } from 'enzyme';

describe('ProfileBuilder', () => {
  let wrapper: any;
  let classes: object;

  beforeEach(() => {
    wrapper = shallow(
      <ProfileBuilder 
        classes={classes}
      />
    );
  });

  it('Matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
