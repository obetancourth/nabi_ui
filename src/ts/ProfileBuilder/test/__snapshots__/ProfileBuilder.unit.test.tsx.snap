// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileBuilder Matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(ProfileBuilder)
    classes={undefined}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "button": "ProfileBuilder-button-1",
        "completed": "ProfileBuilder-completed-2",
        "instructions": "ProfileBuilder-instructions-3",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "button": "ProfileBuilder-button-1",
          "completed": "ProfileBuilder-completed-2",
          "instructions": "ProfileBuilder-instructions-3",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
